tests.R:12:3 [failure] Failure:1. Failure has been forced
tests.R:16:3 [failure] Failure:2a. Failure has been forced
tests.R:19:3 [failure] Failure:2b. FALSE isn't true.
tests.R:24:5 [failure] Failure:loop. `i` not equal to 2.
tests.R:33:3 [error] Error:1. stop
tests.R:47:3 [error] Error:3. !
tests.R:56:3 [error] Recursion:1. This is deep
tests.R:62:3 [skip] Skip:1. Reason: skip
tests.R:69:3 [skip] Skip:2. Reason: skip
tests.R:72:1 [skip] Skip:3. Reason: empty test
tests.R:78:3 [warning] Warning:1. abc
tests.R:84:3 [warning] Warning:2. def
tests.R:85:3 [warning] Warning:2. ghi
tests.R:111:3 [error] Error:4. Throwable
